### ISSUE TITLE
Version 4.1.1 does not appear in apt repo

### ISSUE THREAD

**Issue by aaronmartin1651** [TheHive-Project/TheHive#1890]
Version 4.1.0 appears in the apt repository but 4.1.1 does not.  This means that 4.1.0 broke my installation due to the indexing bug for large fields (typically fields containing hashes).  I would upgrade to 4.1.1 but it is not in apt
after apt update it says no updates available.  I restored to 4.0 using a Commvault image.  After restoring I logged into the server, ran apt-update and then ran `apt list --upgradable`, only 4.1.0 shows as available...

> thehive4/stable 4.1.0-1 all [upgradable from: 4.0.0-1]

**Comment by jeromeleonard** [comment]
Can you please ensure to use the right codename (`release`) of the repository, as specified in the documentation: https://docs.thehive-project.org/thehive/installation-and-configuration/installation/step-by-step-guide/#stable-versions

**Comment by aaronmartin1651** [comment]
This is what we have: 
/etc/apt/sources.list.d/thehive-project.list
`deb https://deb.thehive-project.org stable main`
The documentation appears to call for 
`deb https://deb.thehive-project.org release main`

I made the modification and now it works. Where can we monitor for changes to the repo?  This is pretty important to know.  I only just joined the Discord channel and I suspect it would be part of the announcements there but if there is an official announcements page for important changes like this, I need to add it to my list.
